aider/coders/base_coder.py

```diff
@@ -1455,6 +1474,15 @@ def send_message(self, inp):
                     self.reflected_message = add_rel_files_message
                 return
 
+            tool_results = self.check_for_tool_calls(content)
+            if len(tool_results) > 1:
```

Comment:
Hm, should it be `len(tool_results) > 0`? In case of one MCP server and one tool I don't see a result, because of this condition.